### DISCUSSION TITLE
Create 3 robot does not connect to wifi router

### DISCUSSION THREAD

**Discussion by nagvajara** [iRobotEducation/create3_docs#181] [Questions] [answered]
Earlier Create 3 robots connected to router and ros2 message worked - undock, dock, wall follow ...
Now the robots do not connect to the router. Rebooting the robots and resetting the router didn't solve the problem.
It seems that the robot tries to connect and fails.

**Comment by alsora** [comment]
Hi @nagvajara! Are you able to connect to the robot's webserver and attach here the log that you see there?https://iroboteducation.github.io/create3_docs/webserver/logs/

**Comment by nagvajara** [comment, marked as answer]
I used Ubuntu 20.04 virtual machine. My virtualbox adapter setting was NAT. 
The setting that works is Bridge Adapter.
